marklogic-data-hub/src/main/java/com/marklogic/hub/impl/DataHubImpl.java

```diff
@@ -830,6 +796,9 @@ private void updateModuleCommandList(Map<String, List<Command>> commandsMap) {
         commands.add(loadUserArtifactsCommand);
         commands.add(loadHubArtifactsCommand);
         commands.add(generateFunctionMetadataCommand);
+        if(hubConfig.getIsProvisionedEnvironment()){
```

Comment:
One small tweak here - this new command is specific to InstallIntoDhsCommand, there's no need for DataHubImpl to know about it. Just add it to the list of commands in InstallIntoDhsCommand, just like CopyQueryOptionsCommand.